3.processing_features/scripts/0.merge_sc_cytotable.py

```diff
@@ -77,7 +77,7 @@
         ).resolve(strict=True)),
         "dest_path": str(pathlib.Path(f"{output_dir}/{name}.parquet")),
     }
-    for name in plate_names if not name in pilot_plates  # focus on non-pilot plates
+    for name in plate_names if name in ["Plate_3_prime"]  # focus on Plate_3_prime
```

Comment:
Just confirming if you meant to commit this change (or if it was simply prototyping)

Reply:
Ah I can change this back to avoid confusion! Thanks for noticing!